common/changes/users-petdun-breadcrumbToAriaSpec_2017-05-08-23-51.json

```diff
@@ -0,0 +1,25 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Breadcrumb component now follows aria guidelines for roles and structure\"",
+      "type": "minor"
```

Comment:
There is a prop change although they are both optional to both breadcrumb and IBreadcrumbItem. Still a patch?